cpu/stm32_common/periph/dac.c

```diff
@@ -57,12 +88,8 @@ int8_t dac_init(dac_t line)
 void dac_set(dac_t line, uint16_t value)
 {
     value = (value >> 4);       /* scale to 12-bit */
-    if (dac_config[line].chan) {
-        DAC->DHR12R2 = value;
-    }
-    else {
-        DAC->DHR12R1 = value;
-    }
+
+    DAC_WRITE_OUTPUT(line, value);
```

Comment:
instead of this, I think for now we can just get the number of DAC lines from the CPU module. So I think this we can easily write as:
```c
#ifdef CPU_FAM_STM32F4
   if (dac_config[line].chan) {
        DAC->DHR12R2 = value;
    }
    else {
        DAC->DHR12R1 = value;
    }
#else
    DAC->DHR12R1 = value;
#endif
```
and then drop the macros above